nip47/event_handler_test.go

```diff
@@ -112,17 +125,22 @@ func TestHandleResponse_WithPermission(t *testing.T) {
 		Tags:      nostr.Tags{},
 		Content:   msg,
 	}
+
+	if version != "0.0" {
```

Comment:
I am worried about adding `ifs` in our tests like this for such a critical part. What about we do my suggestion of testing the 4 combinations separately?

Reply:
that or pass the tags in as an argument to this function

Reply:
@im-adithya can you address this and add a new parameter to the function for `tags`?

Reply:
Ah ok. Maybe it's fine as-is then. Thanks!

Reply:
Will also change it here to check `version != ""`